create/managing-nonversioned-and-versioned-cluster-tasks.adoc

```diff
@@ -12,7 +12,7 @@ Both NVCT and VCT have the same metadata, behavior, and specifications, includin
 
 [IMPORTANT]
 ====
-In {pipelines-title} 1.10, cluster task functionality is deprecated and is planned to be removed in a future release.
+In {pipelines-title} 1.10, `ClusterTask` functionality  is deprecated and is planned to be removed in a future release.
```

Comment:
I think OpenShift usually does not format functions or features with backticks (only system items).

Reply:
This is the name of a Custom Resource. But just "cluster task" can also mean the new shiny thing...